install/Knative-custom-install.md

```diff
@@ -0,0 +1,222 @@
+
+| Istio Package Filename | Description       |
+| ---------------------- | ----------------- |
+| [`istio.yaml`][a]             | Default: Configured for Automatic Sidecar Injection. |
+| [`istio-crds.yaml`][b]        | Support for Custom Resource Definitions. **Q: This have to do with "Pilot"? Why choose this?** |
+| [`istio-lean.yaml`][c]        | Configured for Manual Sidecar Injection. **Q: When or why should users choose this?** |
+
+[a]: https://github.com/knative/serving/releases/download/v0.2.2/istio.yaml
+[b]: https://github.com/knative/serving/releases/download/v0.2.2/istio-crds.yaml
+[c]: https://github.com/knative/serving/releases/download/v0.2.2/istio-lean.yaml
+
+### Installing Istio installation packages
+
+1. Install an Istio package with the `kubectl apply` command by specifying the
+   package filename:
+    ```bash
+    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.2/[FILENAME].yaml
+    ```
+
+    where `[FILENAME]` is the name of the Istio package that you want to install. Example:
+    `https://github.com/knative/serving/releases/download/v0.2.2/istio.yaml
+    `
+1. Label the default namespace with `istio-injection=enabled`: **Q: For which install package is this needed? Or is this needed for all Istio installs?**
+    ```bash
+    kubectl label namespace default istio-injection=enabled
+    ```
+1. View the status of your Istio installation. It might take a few
+   seconds so rerun the following command until all of the pods show a
+   `STATUS` of `Running` or `Completed`:
+    ```bash
+    kubectl get pods --namespace istio-system
+    ```
+
+    > Tip: You can append the `--watch` flag to the `kubectl get` commands to
+      view the pod status in realtime. You use `CTRL + C` to exit watch mode.
+
+## Installing Knative components
+
+You can install each Knative component independently or use the default packages
+to install multiple components. The individual component packages exclude
+an observability plugin but you have the option to install one later.
+
+### Choosing Knative installation packages
+
+The following Knative installation packages are available.
+
+This installation method is suggested for experienced operators. If you are new to
+Knative, you should [install the default package](./README.md) to help you get up
+and running with the least effort.
+
+| Knative Package Filename | Serving Component | Build Component | Eventing Component | Observability Plugin* |
+| ------------------------ | ----------------- | --------------- | ------------------ | ----------------- |
+| Default: [`release.yaml`][1] ([knative/serving repo][1.1]) | Included | Included | -     | [ELK stack][2] |
+| [`release-no-mon.yaml`][3] | Included        | Included        | -                  | -                 |
+| [`release-lite.yaml`][4] | Included          | Included        | -                  | Includes support for a [`fluentd` based observability plugin][5] that you provide. |
```

Comment:
@mdemirhan is this an accurate description of what ...-lite.yaml installs or inline with your comment above, does this exclude the ELK stack but add Promeeheus and Grafana for metrics?